editor/src/components/canvas/canvas-strategies/post-action-options/post-action-options.ts

```diff
@@ -66,10 +80,9 @@ export function createPasteToReplacePostActionActions(
     internalClipboard: internalClipboard,
   }
 
-  const defaultChoice = stripNulls([
-    PasteToReplaceWithPropsReplacedPostActionChoice(pasteToReplacePostActionMenuData),
-    PasteToReplaceWithPropsPreservedPostActionChoice(pasteToReplacePostActionMenuData),
-  ]).at(0)
+  const defaultChoice =
+    PasteToReplaceWithPropsReplacedPostActionChoice(pasteToReplacePostActionMenuData) ??
```

Comment:
similarly, I'd rename these to ReplacePropsPasteToReplaceWithPostActionChoice and PreservePropsPasteToReplaceWithPropsActionChoice